src/completion/completeStateNames.ts

```diff
@@ -86,10 +83,7 @@ function getCompletionList(
             item.commitCharacters = [',']
 
             item.kind = CompletionItemKind.Value
-            item.textEdit = TextEdit.replace(
```

Comment:
Text edit is important here. For example, let's say the user wants to change the existing state machine. She can place the cursor on the begining of the text and trigger autocompletion. This change would prevent that. It would also prevent autocompletions when there are empty quotes and the user places cursor between them.

Reply:
If you need to remove it, you could find a way to do it through settings/options that user can provide when asl service is initialized.